client/state/sites/media-storage/selectors.js

```diff
@@ -17,3 +17,17 @@ export function getMediaStorage( state, siteId ) {
 export function isRequestingMediaStorage( state, siteId ) {
 	return !! state.sites.mediaStorage.fetchingItems[ siteId ];
 }
+
+/**
+ * Returns true, if a site is over current plan limits
+ * @param   {Object}  state  Global state tree
+ * @param   {Number}  siteId Site ID
+ * @returns {Boolean}        If site is over storage limits.
+ */
+export function isOverMediaLimit( state, siteId ) {
+	const mediaStorage = state.sites.mediaStorage.items[ siteId ];
+	if ( ! mediaStorage || mediaStorage.max_storage_bytes === -1 ) {
+		return false;
```

Comment:
Is it wise to return `false` when the status is unknown? Instead, should we return `null` as an indicator that overage is unknown?